pkg/kubectl/bind/cmd/cmd.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/spf13/cobra"
 
 	"k8s.io/cli-runtime/pkg/genericclioptions"
```

Comment:
if you narrowly include the exec and oauth plugins instead of the "include all" package, you'll avoid pulling in the deprecated azure and gcp ones that are gone in 1.26, and keep your dep graph much smaller

Reply:
Oh, perfect. Thanks for the hint. Updating.